### PR TITLE
implemented the draw methods and their tests of all Shapes classes

### PR DESCRIPTION
**Pull request by jpgonzales27**

### CODE REVIEW

File: app/src/main/java/src/ShapeL.java [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
+    private static final int ROWS = 2;
+    private static final int COLUMNS = 3;
 
+    /**
+     * Draws the shape L into an array.
+     * @return An array of int with of the form of figure.
+     */
+    @Override
+    public int[][]  draw() {
+        int[][] mat = new int[ROWS][COLUMNS];
```
Comment (JRojas999) [inline app/src/main/java/src/ShapeL.java]: This variable mat is not very descriptive. Maybe we can improve it.

Comment (jpgonzales27) [reply app/src/main/java/src/ShapeL.java]: Thanks, Joel, I'm going to change it now

Comment (jpgonzales27) [reply app/src/main/java/src/ShapeL.java]: It is already changing, I changed the name of the mat variable to matrix

**Review by JRojas999 — APPROVED** [review]

**Review by Raymundogs — APPROVED** [review]

File: app/src/main/java/src/ShapeJ.java [hunk 2]
```diff
@@ -1,9 +1,26 @@
...
+    private static final int ROWS = 2;
+    private static final int COLUMNS = 3;
 
+    /**
+     * Draws the shape J into an array.
+     * @return An array of int with of the form of figure.
+     */
+    @Override
+    public int[][]  draw() {
+        int[][] matrix = new int[ROWS][COLUMNS];
```
Comment (CristelhM) [inline app/src/main/java/src/ShapeJ.java:13]: Shape can have matrix as attribute

Comment (CristelhM) [reply app/src/main/java/src/ShapeJ.java:13]: ShapeJ (){
     // 3x3
     matrix = new [3][3];
}

File: app/src/main/java/src/ShapeL.java [hunk 3]
```diff
@@ -1,9 +1,26 @@
...
+    @Override
+    public int[][]  draw() {
+        int[][] matrix = new int[ROWS][COLUMNS];
+        for (int i = 0; i < matrix.length; i++) {
+            for (int j = 0; j < matrix[0].length; j++) {
+                if (i == 1 && j >= i) {
+                    matrix[i][j] = 0;
+                } else {
+                    matrix[i][j] = 1;
+                }
```
Comment (CristelhM) [inline app/src/main/java/src/ShapeL.java:20]: matrix[i][j] =  (i == 1 && j >= i);

**Pull request merged**